src/main/java/com/SAS/soccer_association_system/stadium/Stadium.java

```diff
@@ -0,0 +1,89 @@
+package com.SAS.soccer_association_system.stadium;
+
+import java.util.LinkedList;
+
+/**
+ * The class represent a soccer stadium of a team
+ */
+public class Stadium {
+
+    private String location;
+    private List<Game> gamesList;
+    private Team homeTeam;
+
+    /**
+     * Empty constructor
+     */
+    public Stadium() {
+        gamesList = new LinkedList<Game>();
+    }
+
+    /**
+     * The method adds a game to the stadium games list
+     * @param location
+     * @param gamesList
+     * @param homeTeam
+     */
+    public Stadium(String location, List<Game> gamesList, Team homeTeam) {
+        this.location = location;
+        this.gamesList = gamesList;
+        this.homeTeam = homeTeam;
+    }
+
+    public boolean addGame(Game newGame){
+        if(newGame == null){
+            return false;
+        }
+
+        gamesList.add(newGame);
+        return true;
+    }
+
+    /**
+     * The function returns the location of the stadium
+     * @return
+     */
+    public String getLocation() {
+        return location;
+    }
+
+    /**
+     * The function sets the location of the stadium
+     * @param location
+     */
+    public void setLocation(String location) {
+        this.location = location;
+    }
+
+    /**
+     * The function returns list of the games in the stadium
+     * @return
+     */
+    public List<Game> getGamesList() {
+        return gamesList;
+    }
+
+    /**
+     * The function sets the game list of the stadium
+     * @param gamesList
+     */
+    public void setGamesList(List<Game> gamesList) {
```

Comment:
I think you may use in the functions "addGame" and "removeGame" instead of the "setGamesList", since we'll probably enter one game at a time and not the whole list I guess.

Reply:
probably will be deleted later if it will not be used